cfgov/cfgov/settings/local.py

```diff
@@ -16,94 +18,89 @@
     },
-    'loggers': {
-        '': {
-            'handlers': ['console'],
-            'level': 'INFO',
-            'propagate': True,
-        }
-    }
 }
 
-# Log database queries.
-if os.environ.get('ENABLE_SQL_LOGGING'):
-    LOGGING['loggers']['django.db.backends'] = {
-        'handlers': ['console'],
-        'level': 'DEBUG',
-        'propagate': False,
+# Log database queries
+if os.environ.get("ENABLE_SQL_LOGGING"):
+    LOGGING["loggers"]["django.db.backends"] = {
+        "handlers": ["console"],
+        "level": "DEBUG",
+        "propagate": False,
     }
 
 # Django Debug Toolbar
-if os.environ.get('ENABLE_DEBUG_TOOLBAR'):
-    INSTALLED_APPS += ('debug_toolbar',)
-
-    MIDDLEWARE_CLASSES += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
+if os.environ.get("ENABLE_DEBUG_TOOLBAR"):
+    INSTALLED_APPS += ("debug_toolbar",)
+
+    if django.VERSION < (2, 0):
+        MIDDLEWARE_CLASSES += (
+            "debug_toolbar.middleware.DebugToolbarMiddleware",
+        )
+        MIDDLEWARE_CLASSES += CSP_MIDDLEWARE
+    else:
+        MIDDLEWARE += ("debug_toolbar.middleware.DebugToolbarMiddleware",)
+        MIDDLEWARE += CSP_MIDDLEWARE
 
     DEBUG_TOOLBAR_PANELS = [
-        'debug_toolbar.panels.versions.VersionsPanel',
-        'debug_toolbar.panels.timer.TimerPanel',
-        'debug_toolbar.panels.settings.SettingsPanel',
-        'debug_toolbar.panels.headers.HeadersPanel',
-        'debug_toolbar.panels.request.RequestPanel',
-        'debug_toolbar.panels.sql.SQLPanel',
-        'debug_toolbar.panels.staticfiles.StaticFilesPanel',
-        'debug_toolbar.panels.templates.TemplatesPanel',
-        'debug_toolbar.panels.cache.CachePanel',
-        'debug_toolbar.panels.signals.SignalsPanel',
-        'debug_toolbar.panels.logging.LoggingPanel',
-        'debug_toolbar.panels.redirects.RedirectsPanel',
-        'flags.panels.FlagsPanel',
-        'flags.panels.FlagChecksPanel',
+        "debug_toolbar.panels.versions.VersionsPanel",
+        "debug_toolbar.panels.timer.TimerPanel",
+        "debug_toolbar.panels.settings.SettingsPanel",
+        "debug_toolbar.panels.headers.HeadersPanel",
+        "debug_toolbar.panels.request.RequestPanel",
+        "debug_toolbar.panels.sql.SQLPanel",
+        "debug_toolbar.panels.staticfiles.StaticFilesPanel",
+        "debug_toolbar.panels.templates.TemplatesPanel",
+        "debug_toolbar.panels.cache.CachePanel",
+        "debug_toolbar.panels.signals.SignalsPanel",
+        "debug_toolbar.panels.logging.LoggingPanel",
+        "debug_toolbar.panels.redirects.RedirectsPanel",
+        "flags.panels.FlagsPanel",
+        "flags.panels.FlagChecksPanel",
     ]
 
     DEBUG_TOOLBAR_CONFIG = {
-        'SHOW_COLLAPSED': True,
-        'SHOW_TOOLBAR_CALLBACK': lambda request: DEBUG,
+        "SHOW_COLLAPSED": True,
+        "SHOW_TOOLBAR_CALLBACK": lambda request: DEBUG,
     }
 
-
-MIDDLEWARE_CLASSES += CSP_MIDDLEWARE_CLASSES
```

Comment:
Why did `CSP_MIDDLEWARE` move up and depend on the check for `ENABLE_DEBUG_TOOLBAR`? Can we move it back down here, or outside of that conditional?

Reply:
That was an accident and I will move it outside of the `ENABLE_DEBUG_TOOLBAR` condition.